lua/hawtkeys/ts.lua

```diff
@@ -342,13 +347,43 @@ local function find_maps_in_file(filePath)
                         rhs = mapping.cmd,
                         from_file = filePath,
                     }
-                    table.insert(tsKemaps, map)
+                    table.insert(tsKeymaps, map)
                 end
             end
         end
     end
 
-    return tsKemaps
+    return tsKeymaps
+end
+
+---@return table
+local function get_keymaps_from_lazy()
+    local lazyKeyMaps = {}
+    for _, args in pairs(hawtkeys.config.keyMapSet) do
+        if args.method == "lazy" then
+            local ok, lazy = pcall(function()
+                return require("lazy").plugins()
+            end)
+            if not ok then
+                error("Lazy Loading requires Lazy")
```

Comment:
Why pcall if you're going to throw an error? I think this should silently fail since we want it to still work for users who aren't using lazy.nvim

Reply:
Good catch

I will update

I will also add some tests for functionality if we don't have Lazy/whichkey